Source/Magick.NET.CrossPlatform/Magick.NET.ubuntu.16.04.vcxproj

```diff
@@ -131,15 +131,21 @@
       <ThreadSafeStatics />
       <RuntimeTypeInfo />
       <AdditionalIncludeDirectories>/usr/local/include/ImageMagick-7;/root/projects/Magick.NET.ubuntu.16.04;/usr/include;$(StlIncludeDirectories);%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
-      <AdditionalOptions>-fPIC %(AdditionalOptions)</AdditionalOptions>
+      <AdditionalOptions>-fPIC -pthread %(AdditionalOptions)</AdditionalOptions>
       <PreprocessorDefinitions Condition="'$(Configuration)|$(Platform)'=='ReleaseQ8|x64'">NDEBUG;MAGICK_NET_LINUX;MAGICKCORE_QUANTUM_DEPTH=8;MAGICKCORE_HDRI_ENABLE=0;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <PreprocessorDefinitions Condition="'$(Configuration)|$(Platform)'=='ReleaseQ16|x64'">NDEBUG;MAGICK_NET_LINUX;MAGICKCORE_QUANTUM_DEPTH=16;MAGICKCORE_HDRI_ENABLE=0;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <PreprocessorDefinitions Condition="'$(Configuration)|$(Platform)'=='ReleaseQ16-HDRI|x64'">NDEBUG;MAGICK_NET_LINUX;MAGICKCORE_QUANTUM_DEPTH=16;MAGICKCORE_HDRI_ENABLE=1;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PositionIndependentCode Condition="'$(Configuration)|$(Platform)'=='ReleaseQ16|x64'">true</PositionIndependentCode>
```

Comment:
Can we remove the `Condition` because this is the same for all the configurations? This is not the only spot where I think we can remove it.